x/subscription/types/keys.go

```diff
@@ -21,18 +24,23 @@ const (
 	// prefix for the subscription fixation store
 	SubsFixationPrefix = "subs-fs"
 
-	// prefix for the subscription fixation store
+	// prefix for the subscription timer store
 	SubsTimerPrefix = "subs-ts"
 
 	// prefix for the CU tracker fixation store
 	CuTrackerFixationPrefix = "cu-tracker-fs"
+
+	// prefix for the CU tracker timer store
+	CuTrackerTimerPrefix = "cu-tracker-ts"
 )
 
-func CuTrackerKey(sub string, provider string, chainID string) string {
-	return sub + " " + provider + " " + chainID
+// CuTrackerKey encodes a keys using the subscription's consumer address, provider address and the relay's chain ID
+func CuTrackerKey(sub string, provider string, chainID string, block uint64) string {
+	return sub + " " + strconv.FormatUint(block, 10) + " " + provider + " " + chainID
```

Comment:
why is the block part of the key?
fixation covers for it in my opinion

Reply:
done. see https://github.com/lavanet/lava/commit/0238aa56ad15f5470c3926934d0a24354947e641 and https://github.com/lavanet/lava/commit/d7935481313f0e116fab2f7cb7b491aef7af8c84